x-pack/plugins/observability/public/config/paths.ts

```diff
@@ -5,8 +5,11 @@
  * 2.0.
  */
 
-export const ALERT_PAGE_LINK = '/app/observability/alerts';
+const APP_BASE = '/app/observability';
```

Comment:
It's better not to define the base path and instead use related utilities to add the base path in a similar way as we do it for other links.

Example: 
https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/pages/cases/cases.tsx#L27

Reply:
+1